homeassistant/components/litterrobot/const.py

```diff
@@ -1,2 +1,13 @@
 """Constants for the Litter-Robot integration."""
+from pylitterbot import FeederRobot, LitterRobot, LitterRobot3, LitterRobot4
+
 DOMAIN = "litterrobot"
+
+
+class SupportedModels(dict):
+    """Supported Litter Robot models."""
+
+    LITTER_ROBOT = LitterRobot
+    LITTER_ROBOT_3 = LitterRobot3
+    LITTER_ROBOT_4 = LitterRobot4
+    FEEDER_ROBOT = FeederRobot
```

Comment:
This seems a bit superfluous. I would probably drop the SupportedModels class and just use the LitterRobot classes directly in `PLATFORMS_BY_TYPE`.